lib/taurus/cli/common.py

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
```

Comment:
Maybe the options implemented for `taurus.cli.taurus_cmd` are good candidates to be also included in `common.py` (e.g. `default_formatter`, `polling_period`,...)

Reply:
I added those in `taurus.cli.common`